src/Acuminator/Acuminator.Vsix/Help Examples/ExampleOfSubMenuDefinition.vsct

```diff
@@ -0,0 +1,99 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we need this file?

Reply:
I thought it could be good as example of vcst definition. There are links to other examples too. Maybe we don't need it though.

Reply:
@dropsonic what do you think?